api/client/network/create.go

```diff
@@ -55,6 +56,7 @@ func newCreateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	flags.StringSliceVar(&opts.labels, "label", []string{}, "Set metadata on a network")
 	flags.BoolVar(&opts.internal, "internal", false, "Restrict external access to the network")
 	flags.BoolVar(&opts.ipv6, "ipv6", false, "Enable IPv6 networking")
+	flags.BoolVar(&opts.manuallyAttachable, "manually-attachable", false, "Enable manual container attachment")
```

Comment:
The reason for the existence of the flag is to make sure that a worker node can run containers only on networks where they were explicitly given permission to do so. The concern is a compromised worker node can compromise security by being able to attach to any network and can do so repeatedly in an effort to deplete network resources and creating a DoS scenario. The way it is designed now, the damage such a compromised node can inflict is only limited to networks which have this flag enabled.